libs/oeth/shared/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I guess this because this becomes a dep of you buildable lib, probably not something we want

Reply:
It is needed if that is buildable. It becomes pretty neat how you can take all the stuff that is exported from that shared library and make it available elsewhere if for some reason you wanted that. We don't actually put any dependencies in this package file though. (I believe)